backend/tracim_backend/applications/agenda/lib.py

```diff
@@ -65,6 +67,12 @@
 """
 
 
+class AgendaCreationState(str, Enum):
```

Comment:
Could you add a comment to explain the differences between `CREATED` and `EXISTING`?

Reply:
done.